react/components/SKUSelector/components/Variation.js

```diff
@@ -54,6 +69,13 @@ const Variation = ({ variation, onSelectItem, maxSkuPrice, isSelected }) => {
               </SelectorItem>
             )
           })}
+          {!showAll && shouldCollapse && (
+            <div className={styles.seeMoreButton}>
+              <Button variation="tertiary" onClick={showAllAction} size="small" collapseLeft>
+                <IOMessage id={seeMoreLabel} values={{ quantity: overflowQuantity }} testId={'seeMoreLabel'} />
```

Comment:
Nice idea!

Use `data-testid` to have a similar API as react testing library https://testing-library.com/docs/dom-testing-library/api-queries#bytestid
```suggestion
                <IOMessage id={seeMoreLabel} values={{ quantity: overflowQuantity }} data-testId="seeMoreLabel" />
```